.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[frontend/*]
```

Comment:
I think it should be `[frontend/**]` according to https://editorconfig.org/

Reply:
You're right, fixed!